### DISCUSSION TITLE
Fig

### DISCUSSION THREAD

**Discussion by hanskokx** [felixse/FluentTerminal#927] [Ideas]
This is so pretty! When can we have it too? 🤣

https://github.com/withfig/autocomplete

![68747470733a2f2f6669672e696f2f676966732f64656d6f2d776974682d6865616465722e676966](https://user-images.githubusercontent.com/1911919/119670113-a018e780-be06-11eb-8add-ccd6bbf231d6.gif)

**Comment by felixse** [comment]
would love to try something like this but I don't have the time right now 😦

**Comment by AnderssonPeter** [comment]
Powershell PSReadLine 2.2+ has a feature that isn't as nice looking but has the somewhat the same functionality.
https://devblogs.microsoft.com/powershell/announcing-psreadline-2-1-with-predictive-intellisense/

Add the following to your profile
```
Set-PSReadLineOption -predictionsource history
Set-PSReadLineOption -EditMode Windows
Set-PSReadLineOption -PredictionViewStyle ListView
```

**Comment by AnderssonPeter** [comment]
@hanskokx if you use powershell then I have started a project that should do this, note its not complete yet and does not know about that many cli tools yet, but its easy to extend, it is functional and currently supports autocompletion for git, npm and adb. 

You can find it [here](https://github.com/AnderssonPeter/PowerType), any feedback is welcome!

**Reply by hanskokx** [reply]
Well, I tried it, but it very much didn't work. I filed a bug.

**Comment by AnderssonPeter** [comment]
@felixse could you pull the suggestions from powershell and display them in a dropdown like fig? (they also contain tooltip information that could be useful)
